src/pages/Thanks/ThanksPageEarthDayChallenge.vue

```diff
@@ -52,7 +52,7 @@
 						<h3 class="tw-text-primary-inverse tw-mr-1.5">
 							You've completed your {{ achievementNumberLanguage }} challenge.
 						</h3>
-						<earth-day-challenge-badge class="tw-h-6 tw-w-6 tw-flex-none tw-mx-auto" />
+						<!-- <earth-day-challenge-badge class="tw-h-6 tw-w-6 tw-flex-none tw-mx-auto" /> -->
```

Comment:
Commenting this out since I removed the svg to avoid import errors. This page will be refactored in a followup pr